bitcoin/src/blockdata/block.rs

```diff
@@ -264,7 +264,7 @@ pub trait BlockCheckedExt: sealed::Sealed {
     fn coinbase(&self) -> Option<&Transaction>;
 
     /// Returns the block height, as encoded in the coinbase transaction according to BIP34.
-    fn bip34_block_height(&self) -> Result<u64, Bip34Error>;
+    fn block_height(&self) -> Result<u64, Bip34Error>;
```

Comment:
Probably worth deprecating mate.

```rust
    fn block_height(&self) -> Result<u64, Bip34Error>;
    
    /// Returns the block height, as encoded in the coinbase transaction according to BIP34.
    #[deprecated(since = "TBD", note = "use `block_height` instead")]
    fn bip34_block_height(&self) -> Result<u64, Bip34Error> { self.block_height() }
```